pkg/replication/replication_mode.go

```diff
@@ -262,7 +263,8 @@ func (m *ModeManager) drSwitchToAsyncWithLock(availableStores []uint64) error {
 		log.Warn("failed to switch to async state", zap.String("replicate-mode", modeDRAutoSync), errs.ZapError(err))
 		return err
 	}
-	dr := drAutoSyncStatus{State: drStateAsync, StateID: id, AvailableStores: availableStores}
+	now := time.Now()
```

Comment:
Do we need to pick it to 6.5 and other branches?

Reply:
need cherry-pick 6.5. i have added the label